pyccel/codegen/printing/fcode.py

```diff
@@ -1193,7 +1195,18 @@ def _print_BooleanFalse(self, expr):
         return '.False.'
 
     def _print_String(self, expr):
-        formatted_str = expr.arg.replace("'","''")
+        # characters speciaux are not handled only newline
+        formatted_str = expr.arg.replace("'","''")\
+                                .replace('\a', '\\a')\
+                                .replace('\b', '\\b')\
+                                .replace('\f', '\\f')\
+                                .replace('\r', '\\r')\
+                                .replace('\t', '\\t')\
+                                .replace('\v', '\\v')
+        if "\n" in formatted_str:
+            s_list = formatted_str.split("\n")
+            code = " // new_line('') // ".join("'{}'".format(s) for s in s_list)
```

Comment:
Does this work if `\n` is at the start of the line?

Reply:
yes, it works.
python ex:
```python
print("\nazul")
```
Fortran code result
```fortran
program prog_sp

implicit none

print *, '' // new_line('') // 'azul'

end program prog_sp
```

Reply:
done